### PR TITLE
Added unit tests for GcloudWrapper.

### PR DESCRIPTION
**Pull request by tmatsuo**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,6 +19,9 @@
         "symfony/browser-kit": "~2|~3",
         "symfony/process": "~2|~3"
     },
+    "require-dev": {
+        "phpunit/phpunit": "~4.0"
```
Comment (bshaffer) [inline composer.json]: Just a thought - you can add [php-cs-fixer](https://github.com/google/google-auth-library-php/blob/master/composer.json#L18) to composer.json and then call it from `vendor/bin` like we do [here](https://github.com/google/google-auth-library-php/blob/master/.travis.yml#L26)

Comment (tmatsuo) [inline composer.json]: Yeah it works too. Now it only downloads php-cs-fixer when needed, so I like it, but I can change it.

Comment (bshaffer) [inline composer.json]: Either way works for me!

File: test/sleeper.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\TestUtils;
+
+// Mock sleep
+
+function sleep($sec)
```
Comment (bshaffer) [inline test/sleeper.php]: no rest for the tested.

**Comment by bshaffer** [comment]
LGTM!

**Comment by tmatsuo** [comment]
Thanks!

**Pull request merged**